### DISCUSSION TITLE
no rules match, no planned actions, but merged, why?

### DISCUSSION THREAD

**Discussion by DrAugus** [Mergifyio/mergify#5071] [Q&A]
![1](https://user-images.githubusercontent.com/34206897/219826097-cc842e09-a500-40db-980a-f80be525d750.png)

pr is [here](https://github.com/DrAugus/draugus.github.io/pull/223)

[mergify rule](https://github.com/DrAugus/draugus.github.io/runs/11427508719)

**Comment by jd** [comment]
If you clicked on details, you'll have the explanation: https://github.com/DrAugus/draugus.github.io/runs/11427508719

<img width="708" alt="CleanShot 2023-02-18 at 11 14 12@2x" src="https://user-images.githubusercontent.com/2644/219854899-cd80bf24-7b3b-4832-a6a0-a27980d85b58.png">

**Reply by DrAugus** [reply]
I saw it, thanks. but what does `another pull request` mean?

**Reply by jd** [reply]
You had another pull request on this repo that included the SHA1 present in your pull request so it got closed by GitHub when the other pull request got merged.
